src/backend/app.js

```diff
@@ -21,10 +21,52 @@ app.use(cors());
 
 router.use("/meals", mealsRouter);
 
+// Homework NJS Week 1
+
+app.get("/future-meals", async (req, res) => {
+  const futureMeals = await knex("homework3.Meal").where(
+    "when",
+    ">",
+    knex.raw("NOW()")
+  );
+  res.json(futureMeals || []);
```

Comment:
How come you have decided to use `|| []` here but not for instance for past meals or all meals?